index.html

```diff
@@ -16,9 +16,25 @@
   <title>Frontend Mentor | Order summary card</title>
 </head>
 <body>
-
-  <!-- START OF BUILD -->
-   
+  <main>
+    <div class="card">
+      <img src="./images/illustration-hero.svg"> 
+    <h1> Order Summary </h1>
+    <p>You can now listen to millions of songs, audiobooks, and podcasts on any device anywhere you like!</p>
+    <div class="card2"> 
```

Comment:
card2 is not a very good class name, can we change this.

Reply:
turned card2 into plan